packages/maker/src/api/defineInterface.ts

```diff
@@ -34,6 +34,7 @@ export function defineInterface(
       string,
       SharedPropertyType | PropertyTypeType
     >;
+    extends?: InterfaceType | InterfaceType[] | string | string[];
```

Comment:
can you add/update test cases in https://github.com/palantir/osdk-ts/blob/main/packages/maker/src/api/overall.test.ts?

Reply:
Done